packages/fxa-content-server/app/scripts/templates/settings/account_recovery/confirm_password.mustache

```diff
@@ -17,7 +17,9 @@
 
             <div class="button-row">
                 <button class="primary-button generate-key-link">{{#t}}Continue{{/t}}</button>
-                <button class="button-link cancel-link">{{#t}}Cancel{{/t}}</button>
+            </div>
+            <div class="links centered">
+              <a href="#" class="cancel-link">{{#t}}Cancel{{/t}}</a>
```

Comment:
I believe all of what you changed should indeed be links except for this one! I double checked our recently created modals in the `fxa-payments-server` package and also `fxa-settings` and we're considering a "cancel" an action (and should be a button).

It looks like we've already got a `primary-button` class and a `secondary-button` class. In case we need to use this later, we could create:

```
.tertiary-button {
  background: transparent;
  color: $blue-60;

  &:hover {
    color: $blue-70;
  }
  &:active {
    color: $blue-80;
  }
}
```

I based these colors on the primary button. Then you can change this back to `button` and add the `button-tertiary` class, but also be sure that margin you added between the primary button and this button is still present.

Reply:
The "cancel" action from events seems to be working with the link, maybe it is not necessary to change to button.

Reply:
Thanks for the review, today I learned something new. :)

I can not find where the `.secondary-button` is implemented. Do you know where it is located?